media-libs/vulkan-layers/vulkan-layers-1.3.239.ebuild

```diff
@@ -37,8 +37,6 @@ DEPEND="${RDEPEND}
 	)
 "
 
-PATCHES="${FILESDIR}/${P}-Build-shared-libs.patch"
-
 multilib_src_configure() {
```

Comment:
also, now we are installing static libraries to the system:

```
-- Installing: /var/tmp/portage/media-libs/vulkan-layers-1.3.239/image/usr/lib64/libVkLayer_utils.a
````

is this what upstream requires?

Reply:
alright thanks, upstream is a joke but i guess we should just listen to them

Reply:
Yeah, weird things often happen in vulkan layers.

Reply:
@Kelvium sorry actually i just thought, technically upstream doesnt like our fix, but is it causing any problems? everything is working as expected now right? or did i miss something?

Reply:
It does work great. The layers work as they did before 1.3.239.

But the author of [this issue](https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/5289) says  shared library "just breaks easier". I've got no clue what they mean by that.

Reply:
ok im gonna discuss this in gentoo-desktop in irc if you want to join, ill post the result here if not

Reply:
ok we decided to remove our patches and just delete the static library in `multilib_src_install_all` since it doesn't need to be installed but is needed for build

also note we still need the `MODULE` -> `SHARED` part of the build shared libs patch to set SONAME correctly

let me know if you need help implementing this